source/browseMode.py

```diff
@@ -552,6 +552,19 @@ def script_disablePassThrough(self, gesture):
 		reportPassThrough(self)
 	script_disablePassThrough.ignoreTreeInterceptorPassThrough = True
 
+	def _set_disableAutoPassThrough(self, state):
+		if (
+			state
+			and not config.conf["virtualBuffers"]["autoFocusFocusableElements"]
+			and self.passThrough

```

Comment:
I think some comments here would help. It seems backwards that `self._focusLastFocusableObject()` is called when `config.conf["virtualBuffers"]["autoFocusFocusableElements"]` is False